src/footprints/index.js

```diff
@@ -17,4 +17,5 @@ module.exports = {
     slider: require('./slider'),
     trrs: require('./trrs'),
     via: require('./via'),
-}
\ No newline at end of file
+    tentingpuck: require('./tentingpuck'),
```

Comment:
nitpick: If you put this above `trrs` this will remain alphabetically sorted

Reply:
Ooh, good point.